ui2/src/common/FormFields.jsx

```diff
@@ -30,3 +30,7 @@ export const SelectField = ({ field: { name }, options }) => (
     </Select>
   </Form.Item>
 );
+
+export const AppInput = ({ placeholder, onChange, value }) => (
+  <Input placeholder={placeholder} onChange={onChange} value={value} />
```

Comment:
looks even more redundant

Reply:
yes, i know, thought I might go ahead and wrap the antd... I'll remove this